tests/bluetooth/host/keys/bt_keys_get_addr/src/main.c

```diff
@@ -11,11 +11,25 @@
 #include "mocks/conn.h"
 #include "mocks/hci_core.h"
 #include "mocks/keys_help_utils.h"
-#include "host_mocks/print_utils.h"
 #include "testing_common_defs.h"
+#include "common/bt_to_str.h"
+#include <stdio.h>
```

Comment:
Don't we usually have the standard includes at the top?

Reply:
Fixed.

Reply:
Yes, we should be consistent about this, but note that this will likely change in the future. Header ordering was discussed in yesterday's dev meeting and will likely come to the architecture meeting next week or in the near future. 

See #41543 and https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes which is the style that may eventually be adopted by Zephyr.